src/server/main_service.cc

```diff
@@ -840,6 +840,7 @@ void Service::DispatchCommand(CmdArgList args, facade::ConnectionContext* cntx)
   intrusive_ptr<Transaction> dist_trans;
 
   bool dispatching_in_multi = under_script || under_multi;
+  string_view cmd_name{cid->name()};
```

Comment:
```suggestion
  const string_view cmd_name{cid->name()};
```

Reply:
This is no longer needed since we have removed the explicit handling.